### PR TITLE
release notes for 1.0

### PR DESCRIPTION
**Pull request by kailuowang**

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,6 +1,18 @@
+## Version 1.0.0
+
+> 2017 Dec 18
```
Comment (tpolecat) [inline CHANGES.md]: Confusion. Shouldn't this be Dec 25?

Comment (kailuowang) [reply CHANGES.md]: oops, corrected.

File: CHANGES.md [hunk 2]
```diff
@@ -1,6 +1,18 @@
+## Version 1.0.0
+
+> 2017 Dec 25
+
+### documentations
```
Comment (tpolecat) [inline CHANGES.md]: In prior releases this section is titled **Documentation Improvements/Additions:**

Comment (kailuowang) [reply CHANGES.md]: fixed.

**Pull request merged**